recipes/pylibdmtx/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pylibdmtx" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: "https://github.com/NaturalHistoryMuseum/pylibdmtx.git"
+  git_rev: "v{{ version }}"
```

Comment:
We strongly discourage git urls. Can you ping upstream for a release?

Reply:
Updated not use use git_url. Added build.sh and bld.bat to install from wheel files available on pypi.org